src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.js

```diff
@@ -58,16 +58,20 @@ class DeleteConfirmation extends React.Component<Props> {
                 </div>
                 <div>
                     <Button
+                        aria-label={commonMessages.cancel.defaultMessage}
                         className="bcs-DeleteConfirmation-cancel"
                         onClick={onDeleteCancel}
+                        data-testid="bcs-delete-confirmation-cancel"
```

Comment:
I think this is not needed in light of the label. Removing it will help push people towards selecting by label :)